stock_picking_warn_message/models/stock_picking.py

```diff
@@ -8,8 +8,28 @@ class StockPicking(models.Model):
 
     _inherit = "stock.picking"
 
+    picking_warn = fields.Text(compute="_compute_picking_warn")
```

Comment:
It seems you change more than the color.

You should explain why you changed that standard behaviour.

Reply:
Hi @rousseldenis, thanks for your feedback, to be able to distinguish between a warning and a blocking message (to be able to show it with a different color) this computed field is used. As some logic is involved in the calculation, I guess it wouldn't make sense to put all that into the view.
Apart from the color the behaviour shouldn't change here as far as I can see (though in a later improvement it might make sense to also block the completion of the picking when there is a blocking message).